app/backend/src/tests/test_utils.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from timezonefinder import TimezoneFinder
+
+from tests.test_fixtures import testconfig
+
+
+@pytest.fixture(autouse=True)
+def _(testconfig):
+    pass
+
+
+def test_to_timezone_lng_lat():
+    tf = TimezoneFinder()
+    lat, lng = 52.5061, 13.358
+    assert tf.timezone_at(lng=lng, lat=lat) == "Europe/Berlin"
```

Comment:
You need to test the actual function in `utils.py`, not the TimezoneFinder module (I hope they have their own tests!)